### DISCUSSION TITLE
Create Issue in a Repo via Email

### DISCUSSION THREAD

**Discussion by jeremy-farrance** [community/community#5866] [Projects and Issues]
This is not an original idea, you can currently do this in Asana. 
 
Here is a quick recap of [what happens in Asana](https://asana.com/guide/help/email/email-to-asana)... Each project (think repo), they provide a custom email address that you can send an email to. It will then create a Task in the Project.
 
Things it intelligently converts:
- the email Subject become the Task title
- the email Body becomes the Task desc
- an additional email in the To field, if it matches an Asana user, assigns the new Task to that user

This seems highly useful in many GitHub scenarios. The simplicity of hitting a unique email address with an intelligent conversion process seems (to me) to have an impressive ROI.
 
Imagine getting a bug report via email, you click Forward to gh+3014479146883@automail.github.io (a demonstrative, but completely fabricated email address by me just now) which you have save in your Contacts under the name "MyProject OnGitHub" so its easy to find and reuse.
 
GitHub receives it and 2 to 5 seconds later the Issue is in your Repo/Project, maybe auto-added to backlog. Notifications go out to a person or two so they can review and clean it up, add stuff...
 
Sound like (to me) the type of magic that has huge ROI.

**Comment by patrickwolf** [comment]
Yes this is needed as Projects (Beta) is being rolled out. I'm surprised it doesn't have more traction
